grammar/en/src/rules.rs

```diff
@@ -1366,18 +1487,23 @@ pub fn rules_time(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
                 };
                 Ok(helpers::hour(hour, true)?.precision(Approximate))
     });
-    b.rule_2("<time-of-day> sharp",
+    b.rule_2("<time-of-day> approximately",
              time_check!(form!(Form::TimeOfDay(_))),
-             b.reg(r#"(?:sharp|exactly)"#)?,
-             |time, _| Ok(time.value().clone().not_latent().precision(Precision::Exact))
+             b.reg(r#"(?:-?ish|about|around|approximately)"#)?,
```

Comment:
Does that mean we are now accepting datetimes like "9pm about" and "9pm around" ?

Reply:
yes

Reply:
Isn't it a bit too loose as a rule? This can end up in false approximative precision in sentences like "Meeting at 8am about languages", where the datetime is exact but will be considered approximate.

Reply:
Could be, indeed. We could remove "about" and "around" from this rule.

Reply:
👍

Reply:
Done and pushed